rtcclient/base.py

```diff
@@ -17,12 +17,17 @@ class RTCBase(object):
     OSLC_CR_XML = "application/x-oslc-cm-change-request+xml"
     OSLC_CR_JSON = "application/x-oslc-cm-change-request+json"
 
-    def __init__(self, url):
+    def __init__(self, url, **kwargs):
```

Comment:
Shall we use `__init__(self, url, skip_full_attributes=True, **kwargs)` instead

Reply:
Done